homeassistant/components/weatherflow_cloud/strings.json

```diff
@@ -23,5 +23,73 @@
       "already_configured": "[%key:common::config_flow::abort::already_configured_service%]",
       "reauth_successful": "[%key:common::config_flow::abort::reauth_successful%]"
     }
+  },
+  "entity": {
+    "sensor": {
+      "air_temperature": {
+        "name": "[%key:component::sensor::entity_component::temperature::name%]"
+      },
+      "air_density": {
+        "name": "Air density"
+      },
+      "barometric_pressure": {
+        "name": "Pressure barometric"
+      },
+      "sea_level_pressure": {
+        "name": "Pressure sea level"
+      },
+      "station_pressure": {
+        "name": "Pressure station"
```

Comment:
pressure what?

Reply:
pressure at the weather station... but I don't know what the use case is honestly -> I'm going to remove it.

Reply:
> station pressure is the pressure as measured at your location, but weatherwise that isn’t as useful as the sea level pressure, which is your pressure but corrected for sea level. That makes it possible to compare pressure values independently from your elevation.

- Station or Gauge Pressure (key pressure): This is the absolute, raw pressure as measured by your instrument. It is not corrected for altitude or pressure. Pilots call this QFE.

I think I'll pull it and we can add it later.